### DISCUSSION TITLE
Reply for (thank you) in different languages

### DISCUSSION THREAD

**Discussion by mohamedsabil83** [Laravel-Lang/common#20] [Q&A]
Lang | You're Welcome
-- | --
ar | العفو / لا شكر على واجب
az |  
be |  
bg |  
bn | 
bs |  
ca |  
cs |  
cy |  
da |  
de | 
de_CH | 
el |  
es | 
et |  
eu |  
fa |  
fi |  
fil |  
fr | 
gl |  
he |  
hi |  
hr |  
hu |  
hy |  
id |  
is |  
it | 
ja | 
ka |  
kk |  
km |  
kn |  
ko |  
lt |  
lv |  
me |  
mk |  
mn |  
mr |  
ms |  
nb |  
ne |  
nl | 
nn |  
pl | 
ps |  
pt | 
pt_BR | 
ro |  
ru | Пожалуйста / Всегда пожалуйста / Обращайтесь
sc |  
si |  
sk |  
sl |  
sq |  
sr_Cyrillic |  
sr_Latin |  
sv |  
sw |  
tg |  
th |  
tk |  
tl |  
tr | 
ug |  
uk |  
ur |  
uz_Cyrillic |  
uz_Latin |  
vi |  
zh_CN |  
zh_HK |  
zh_TW

**Comment by andrey-helldar** [comment]
Great table! Do you think it's worth concatenating it with https://github.com/Laravel-Lang/lang/discussions/1421 so that the whole table is in one place?

**Reply by mohamedsabil83** [reply]
Yep, I thought like that when decide to add it but forget today and made a new one 😄

**Reply by andrey-helldar** [reply]
Get up, got up, but forgot to wake up 😂

**Reply by andrey-helldar** [reply]
Then I will merge the tables into one. Thank you!

**Reply by andrey-helldar** [reply]
Ready https://github.com/Laravel-Lang/lang/discussions/1421 :)

**Reply by andrey-helldar** [reply]
شكراً لك

**Reply by mohamedsabil83** [reply]
Пожалуйста 😎

**Reply by biskukuy** [reply]
es: muchas gracias

**Reply by andrey-helldar** [reply]
@biskukuy, the main table is [here](https://github.com/Laravel-Lang/lang/discussions/1421). I added your suggestion to it. Muchas gracias 🤝
